plugins/plugins.gradle.kts

```diff
@@ -10,17 +10,18 @@ subprojects {
         }
         val sourceJar = task<Jar>("sourceJar") {
             description = "Creates a JAR that contains the source code."
-            from(java.sourceSets["main"].allSource)
+            from(project.sourceSets["main"].allSource)
             classifier = "sources"
         }
+        val javaPluginConvention = convention.plugins["java"] as JavaPluginConvention
```

Comment:
@JLLeitschuh This is ugly as hell, but I couldn't find a better way to get the output docs directory without hardcoding it